src/lib/wsman-curl-client-transport.c

```diff
@@ -283,7 +283,10 @@ init_curl_transport(WsManClient *cl)
 	CURLcode r = CURLE_OK;
         char *sslhack;
         long sslversion;
-        dictionary *ini = iniparser_new(wsmc_get_conffile(cl));
+        dictionary *ini = NULL;
+#ifndef _WIN32
+        ini = iniparser_new(wsmc_get_conffile(cl));
```

Comment:
why is the initialization ifdef'ed out for Windows ?

Reply:
The commit title should actually explain it:  "Hide wsmc_get_conffile behind _WIN32 (like prototype)". With commit 9ec439aa18efa48cce3eaa999f9b9742795ee60b this line was introduced without changing the wsmc_get_conffile prototype to be available with _WIN32. From there, I do not think the initialization compiled on Windows. This should be fixed but I do not know your configuration architecture and do not know why the .h has the ifdefs as they are.

Reply:
But where do you call `iniparser_new` now ?

Reply:
Nowhere. This makes the curl client transport compile on Windows (or cross compile for it) but does not fix the initialization which did not even compile before. So even though this is still non-functional it at least compiles.